src/dndbeyond/content-scripts/character.js

```diff
@@ -1160,7 +1160,7 @@ function handleSpecialSpells(spell_name, damages=[], damage_types=[], {spell_sou
             break;
         }
     }
-    if (elementalAffinity && damage_types.includes(elementalAffinity)) {
+    if (elementalAffinity && damage_types.includes(elementalAffinity.toLowerCase())) {
```

Comment:
`damage_types` would have the non lower case type, I believe, so checking if the lowercase elemental affinity is included in `damage_types`  (which wouldn't be lower case) might fail.
Solution would have to be `damage_types.some(type => type.toLowerCase() === elementalAffinity.toLowerCase())`